lib/task.py

```diff
@@ -294,7 +294,7 @@ def join(self):
                 event = Event()
                 event.ttl = self.ttl * self.ttl_multiplier
                 event.tags = self.tags
-                event.attributes = self.attributes
+                event.attributes = dict(self.attributes)
```

Comment:
As it turns out, this will be a shallow copy. This is technically OK in our case, since Riemann does not support compound objects as attributes. However, we can make this a deep copy if that feels more "complete". Let me know.

```python
Python 2.7.11 (default, Dec 22 2015, 22:24:43)
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> d1 = {"foo": "bar"}
>>> d2 = d1
>>> d3 = dict(d1)
>>> d1.update({"fizz": "buzz"})
>>> d1
{'foo': 'bar', 'fizz': 'buzz'}
>>> d2
{'foo': 'bar', 'fizz': 'buzz'}
>>> d3
{'foo': 'bar'}
>>>
```